scipy/optimize/optimize.py

```diff
@@ -157,7 +157,7 @@ def is_array_scalar(x):
 def vecnorm(x, ord=2):
     if ord == Inf:
         return numpy.amax(numpy.abs(x))
-    elif ord == -Inf:
+    elif ord == -(Inf):
```

Comment:
TBH I'm -1 for this change.

Reply:
Yeah. That's okay. I have just made it to be on safe side.

Reply:
I agree. Thanks though @urstrulykkr. Just take your time getting familiar with SciPy and finding something that interests you. No need to make such tiny fix PRs for GSoC requirements.